SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/Utils/DataTypeUtilities.cpp

```diff
@@ -53,7 +53,10 @@ FString AlphanumericSanitization(const FString& InString)
 FString UnrealNameToSchemaComponentName(const FString& UnrealName)
 {
 	FString SchemaTypeName = UnrealNameToSchemaName(UnrealName);
-	SchemaTypeName[0] = FChar::ToUpper(SchemaTypeName[0]);
+	if (!SchemaTypeName.IsEmpty())
+	{
+		SchemaTypeName[0] = FChar::ToUpper(SchemaTypeName[0]);
+	}
```

Comment:
If this is empty would we want to error?

Reply:
We will error in `CheckIdentifierNameValidity` (which calls this).